### PR TITLE
LPS-142557 Collection Display for Objects does not display all instances, even when "Display All Collection Items" is checked 

### PR DESCRIPTION
**Pull request by ruben-pulido**
//cc @ealonso

### CODE REVIEW

**Comment by ruben-pulido** [comment]
ci:test:sf

**Comment by ruben-pulido** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/layout-data-items/Collection.js [hunk 1]
```diff
@@ -382,10 +382,12 @@ const Collection = React.memo(
 						collectionConfig={collectionConfig}
 						collectionId={item.itemId}
 						onPageChange={setActivePage}
+						showAllItems={
```
Comment (sandrodw3) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/layout-data-items/Collection.js:385]: Do we need to add this param if we are already passing `collectionConfig`? 

Maybe we could calculate inside `CollectionPagination`, what do you think?

**Review by sandrodw3 — COMMENTED** [review]
Looks super cool! 🥰
I just left two tiny comments!

**Pull request closed without merging**